src/FilterList.js

```diff
@@ -153,18 +156,18 @@ export let Label = _.flow(
     </Flex>
   )
 })
-Label.displayName = 'Label'
 
-export let FieldLabel = contexturify(
-  ({ tree, node, node: { field } = {}, fields, label }) => (
-    <Label tree={tree} node={node} fields={fields}>
-      {label || _.get([field, 'label'], fields) || field}
-    </Label>
-  )
-)
-FieldLabel.displayName = 'FieldLabel'
+export let FieldLabel = _.flow(
+  setDisplayName('FilterLabel'),
+  contexturify
+)(({ tree, node, node: { field } = {}, fields, label }) => (
+  <Label tree={tree} node={node} fields={fields}>
+    {label || _.get([field, 'label'], fields) || field}
+  </Label>
+))
 
 export let FilterList = _.flow(
+  setDisplayName('FilterList'),
```

Comment:
We've got to do this before the HOCs (`withTheme`, etc...) wrap the component, otherwise they can't pick up on the wrapped component's name.